phpstan.neon.dist

```diff
@@ -2,58 +2,58 @@ parameters:
     phpVersion: 80100
     level: 3
     paths:
-        - lib
+        - src
         - tests
     excludePaths:
-        - lib/vendor/doctrine-build-common
-        - tests/Doctrine/Tests/Common/Proxy/InvalidReturnTypeClass.php
-        - tests/Doctrine/Tests/Common/Proxy/InvalidTypeHintClass.php
-        - tests/Doctrine/Tests/Common/Proxy/LazyLoadableObjectWithTypedProperties.php
-        - tests/Doctrine/Tests/Common/Proxy/MagicIssetClassWithInteger.php
-        - tests/Doctrine/Tests/Common/Proxy/NullableNonOptionalHintClass.php
-        - tests/Doctrine/Tests/Common/Proxy/PHP81NeverType.php
-        - tests/Doctrine/Tests/Common/Proxy/PHP81IntersectionTypes.php
-        - tests/Doctrine/Tests/Common/Proxy/Php8UnionTypes.php
-        - tests/Doctrine/Tests/Common/Proxy/Php8StaticType.php
-        - tests/Doctrine/Tests/Common/Proxy/ProxyGeneratorTest.php
-        - tests/Doctrine/Tests/Common/Proxy/ProxyLogicTypedPropertiesTest.php
-        - tests/Doctrine/Tests/Common/Proxy/SerializedClass.php
-        - tests/Doctrine/Tests/Common/Proxy/VariadicTypeHintClass.php
-        - tests/Doctrine/Tests/Common/Proxy/Php71NullableDefaultedNonOptionalHintClass.php
-        - tests/Doctrine/Tests/Common/Proxy/generated
+        - src/vendor/doctrine-build-common
```

Comment:
what is this path ?

Reply:
I'll make a separate PR to get rid of it, it should have been removed in 4167060592de4dce2b955c6b6dded9a1b43a04b9 I think

Reply:
#976